authentication/DjangoAuthentication.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * A script to provide authentication for Django apps.
+ *
+ * First it makes a GET request and obtains the csrfmiddlewaretoken from the response body.
+ *
+ * Then it makes a POST request with a body which contains username, password and csrfmiddlewaretoken.
+ *
+ * A successful login will result in a 302 redirect. If this happens, a GET request is made to the redirect URL.
+ *
+ * Every request made by this script is logged seperately to the History tab.
```

Comment:
There's a typo in separately.

Reply:
I think I'm destined to always misspell that word!

What's the best way for me to fix it? Should I amend the commit and force push, or make the correction in a new commit?

Reply:
Amend and force is best, to maintain an atomic commit.
https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md#what-we-zap-team-expect-from-you

Reply:
For sure. Thanks for drawing my attention to the contributing document.

I just force pushed then.